src/actions/document.js

```diff
@@ -1,5 +1,17 @@
 export const UPDATE_MARKDOWN = 'UPDATE_MARKDOWN';
 export const updateMarkdown = markdown => ({
   type: UPDATE_MARKDOWN,
-  payload: { markdown }
+  payload: markdown
+});
+
+export const UPDATE_FILES = 'UPDATE_FILES';
+export const updateFiles = file => ({
+  type: UPDATE_FILES,
+  payload: file
+});
+
+export const UPDATE_SELECTED_FILE = 'UPDATE_SELECTED_FILE';
```

Comment:
The card just asks to display a list of markdown files. this seems outside the scope of that.